mergify_engine/gh_pr.py

```diff
@@ -122,6 +122,11 @@ def mergify_engine_merge(self, rule):
         # to repoduce the issue
 
 
+def base_is_modifiable(self):
+    return (self.raw_data["maintainer_can_modify"] or
```

Comment:
nit: `self.maintainer_can_modify`

Reply:
Is this exposed? I removed the attribute from the existing code since it was not used anywhere else.

Reply:
Oh I see my bad, this is clearly missing from the pygithub